frontend/src/locales/scripts/en.json5

```
@@ -50,9 +50,9 @@
     aboutTab: "About",
     resourcesTab: "Resources",
     loading: "Loading...",
-    filterButton: {
+    filters: {
```

Just a note that by changing the name of this key, it will cause the translations for `filterButton.simple` to also go away and need to be translated again. It's better to keep the key the same, if we can, to reduce churn on the translators. This also has an immediate effect on translations in production, any languages that had `filterButton.simple` translated will no longer have it translated.

Ideally, we don't change the keys, if avoidable, and in this case it is (even if it's annoying to have the overly specific name).

Suggesting we keep the key the same for these reasons (even though it wasn't a great name to start with):

```suggestion
    filterButton: {
```